docs/getting_started.rst

```diff
@@ -11,81 +11,81 @@ edx-platform. It provides new tracking backends and processors.
-                            "regular_expressions": [
-                                "edx.course.enrollment.*",
-                                "seek_video",
-                                "edx.video.position.changed"
+                        'ENGINE': 'eventtracking.processors.regex_filter.RegexFilter',
+                        'OPTIONS': {
+                            'filter_type': 'allowlist',
+                            'regular_expressions': [
+                                'edx.course.enrollment.*',
+                                'seek_video',
+                                'edx.video.position.changed'
                             ]
                         }
                     }
                 ],
-                "backends": {
-                    "caliper": {
-                        "ENGINE": "event_routing_backends.backends.events_router.EventsRouter",
-                        "OPTIONS": {
-                            "processors": [
+                'backends': {
+                    'caliper': {
+                        'ENGINE': 'event_routing_backends.backends.events_router.EventsRouter',
+                        'OPTIONS': {
+                            'processors': [
                                 {
-                                    "ENGINE": "event_routing_backends.processors.caliper.transformer_processor.CaliperProcessor",
-                                    "OPTIONS": {}
+                                    'ENGINE': 'event_routing_backends.processors.caliper.transformer_processor.CaliperProcessor',
+                                    'OPTIONS': {}
                                 },
                                 {
-                                    "ENGINE": "event_routing_backends.processors.caliper.envelop_processor.CaliperEnvelopProcessor",
-                                    "OPTIONS": {
-                                        "sensor_id": "http://test.com/sensors"
+                                    'ENGINE': 'event_routing_backends.processors.caliper.envelop_processor.CaliperEnvelopProcessor',
+                                    'OPTIONS': {
+                                        'sensor_id': 'http://example.com/sensors'
                                     }
                                 }
                             ],
-                            "backend_name": "caliper"
+                            'backend_name': 'caliper'
                         }
                     }
                 }
             }
         }
     }
 
-- Run migrations
+- Run migrations in lms-shell
    .. code-block:: bash
 
     $ ./manage.py lms migrate event_routing_backends
 
-- Add router configuraton from django admin under ``EVENT_ROUTING_BACKENDS`` section
+- Add router configuraton from django admin under ``EVENT_ROUTING_BACKENDS`` section (http://localhost:18000/admin/event_routing_backends/routerconfiguration/add/) using backend name ``caliper``
```

Comment:
For reviewer: I am unsure if this backend name is correct.

Reply:
Its correct